README.md

```diff
@@ -58,6 +65,9 @@ SSZipArchive requires ARC.
 ### Swift
 
 ```swift
+//Import
+import ZipArchive //import SSZipArchive still works for previous code
```

Comment:
Are you sure that both import ZipArchive and import SSZipArchive will work with our latest version?

Reply:
So best I can tell, is it won't work if you pull it in via SPM, but if you had SSZipArchive before via another method it works. I tried both and both build, but then again, both have always built for me.

Reply:
Another option would be to just announce a possible breaking change and push towards ZipArchive for all. Thoughts?

Reply:
If you do a clean pod reinstall, I don't think both syntaxes will work.
The comment might lead to people reporting import issues.
Can you try `pod deintegrate && pod install`?
- if `import ZipArchive` works after that, then probably just leave "import ZipArchive" in the doc.
- If `import SSZipArchive` is required, the specify "import ZipArchive" for SPM/Carthage, and "import SSZipArchive"  for CocoaPods.

Same thing for ObjC.

Reply:
You are right, I created a fresh project and pods does require SSZipArchive. I'll match your language.

Reply:
Ok, language updated.